### ISSUE TITLE
Java driver fails to parse big generated file

### ISSUE THREAD

**Issue by smola** [bblfsh/java-driver#102]
* Version: `bblfsh/bblfshd:2.11.0-drivers`
* Possibly related to: #88
* Parsing the following file: https://raw.githubusercontent.com/apache/spark/master/sql/hive-thriftserver/src/gen/java/org/apache/hive/service/cli/thrift/TCLIService.java

bblfshd logs:
```
time="2018-12-13T12:01:54Z" level=error msg="request processed content 516093 bytes error: rpc error: code = Internal desc = Unexpected character ('c' (code 99)): was expecting comma to separate Object entries
 at [Source: java.io.StringReader@78aab498; line: 1, column: 65540]" elapsed=214.176945ms language=java
```

Maybe the JSON request i or response are too long to be serialized in one line?

**Comment by smola** [comment]
When running this in a loop, I'm also getting:

```
time="2018-12-13T12:01:54Z" level=error msg="request processed content 516093 bytes error: rpc error: code = Internal desc = read |0: i/o timeout" elapsed=5.001223945s language=java
```

**Comment by dennwc** [comment]
Fixed in `bblfsh/bblfshd:v2.12.0-drivers`.
